Sources/URLRouting/Printing.swift

```diff
@@ -51,7 +51,7 @@ extension ParserPrinter where Input == URLRequestData {
 /// let authenticatedRouter = router
 ///   .baseRequestData(.init(headers: ["X-PointFree-Session": ["deadbeef"]]))
 ///
-/// try authenticatedRouter.request(for: .home)
+/// try authenticatedRouter.response(for: .home)
```

Comment:
should this actually be something more like this?

```suggestion
/// try authenticatedRouter.decodedResponse(for: .home)
///   .response
```

Reply:
This was an accidental rename in the docs. It's the API for getting a URLRequest from a router.

Reply:
ah coo